firefox-ios/firefox-ios-tests/Tests/XCUITests/ToolbarMenuTests.swift

```diff
@@ -48,11 +48,11 @@ class ToolbarMenuTests: BaseTestCase {
         mozWaitForElementToExist(hamburgerMenu)
         mozWaitForElementToExist(app.textFields[AccessibilityIdentifiers.Browser.AddressToolbar.searchTextField])
         mozWaitForElementToExist(tabsButton)
+        mozWaitForElementToExist(firstPocketCell)
         XCTAssertTrue(
             hamburgerMenu.isRightOf(rightElement: tabsButton),
             "Menu button is not on the right side of tabs button"
         )
-        mozWaitForElementToExist(firstPocketCell)
```

Comment:
When the device is getting rotated, the element exist but may need more time to get to the positions.